flink-end-to-end-tests/flink-stream-state-ttl-test/pom.xml

```diff
@@ -50,6 +44,7 @@ under the License.
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-statebackend-rocksdb_2.11</artifactId>
```

Comment:
You said this dependency is unnecessary. I concurr.

Reply:
missed that one, removed!